SPECIFICATION.md

```diff
@@ -0,0 +1,143 @@
+first character on every line. The number sign is followed by a space, a key,
+and a value. Key and value are separated by a colon (`:`). Everything following
+the colon is considered the value. Both keys and values are strings.
+
+Example:
+
+    # Format: FontoBene/1.0
+    # Creator: Max Müller
+    # Version: 1.0
+    # VersionDate: 2017-09-27
+    # License: GPLv3+
+
+All keys that may be used are specified in the specification. A user MAY use
+non-standard keys, but must prefix them with `X-`, for example `X-Foo: Bar`.
+
+Whitespace around key and value should be stripped by the parsers.
+
+
+### Body
+
+The first empty line separates the header from the body. The body consists of
+glyph definition blocks separated by at least one empty line.
+
+Every glyph definition starts with the glyph declaration, a unicode codepoint
+in square brackets (e.g. `[1F4A9]` for the pile of poo character). It may
+optionally be followed by a UTF-8 representation of the character for easier
+previewing (e.g. `💩`). However, that preview will be ignored by the parser.
+
+The notation used inside the brackets follows the convention described in *"The
+Unicode Standard, Version 10.0, Appendix A, Notational Conventions, Code
+Points"*:
+
+> In running text, an individual Unicode code point is expressed as U+n, where
+> n is four to six hexadecimal digits, using the digits 0–9 and uppercase
+> letters A–F (for 10 through 15, respectively).
+>
+> Leading zeros are omitted, unless the code point would have fewer than four
+> hexadecimal digits—for example, U+0001, U+0012, U+0123, U+1234, U+12345,
+> U+102345.
+>
+> The U+ may be omitted for brevity in tables or when denoting ranges.
+
+The glyph declaration is followed by a list of polylines or glyph references,
+each on a single line.
+
+A polyline looks like this:
+
+    0,0;3,9;6,0
+
+It consists of coordinate pairs, separated by semicolons (`;`). The X and Y
+coordinates are separated with a comma (`,`).  The coordinates are arbitrary
+precision floating point numbers using the full stop (U+002E) as decimal mark.
+If a coordinate is a whole number, the decimal mark and the fractional part of
+the number may be omitted.
+
+The coordinate system starts at the lower left corner. Positive X coordinates
+point to the right. Positive Y coordinates point to the top. Negative
+coordinates are not allowed.
+
+          Y
+    (0,Y) +
+          |
+          |
+          +-----+ X
+      (0,0)   (X,0)
+
+The example above would draw the pointy "outer frame" of the character "A".
+Here is the complete definition of the glyph for the character "LATIN CAPITAL
+LETTER A (U+0041)":
+
+    [0041] A
+    0.8333,2.5;5.1666,2.5
+    0,0;3,9;6,0
+
+Because many glyphs are very similar, references may be used as "includes".
+They start with the "@" symbol (U+0040), followed by a codepoint. Here is the
+definition of the glyph for the character "LATIN CAPITAL LETTER A WITH
+CIRCUMFLEX (U+00C2)":
```

Comment:
Should we use the "@" prefix for includes, or stick with "C", or use something else?

Reply:
I would stick with "C" for the moment because it's a "simpler" character than "@" and because of the better compatibility to LFF. We could still open an issue some time to discuss about changing that character ;)